LSTM_model.py

```diff
@@ -43,14 +31,14 @@
 reco_rate_Dc = '/recommendation_requests_5m_rate_dc'
 
 paths_cross_dc = [[avg_cpu_load_DC, 'avg_cpu_load'], [avg_heap_DC, 'avg_heap'], [avg_memory_Dc, 'avg_memory']
```

Comment:
The load part is too specific, please write a method that will load the files without specifying their names :)